html-relative-links/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="/html-relative-links/index.html">Home Page</a></li>
+    <li><a href="/html-relative-links/about-me/about-me.html">About Me</a></li>
+    <li><a href="/html-relative-links/projects/projects.html">Projects</a></li>
+    <li><a href="/html-relative-links/contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
you can do without `/html-relative-links/` in your paths here since these are all children of the current directory.